### PR TITLE
i3status-rust: update to 0.31.5.

### PR DESCRIPTION
**Pull request by inalone**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/i3status-rust/template [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 pkgname=i3status-rust
-version=0.22.0
-revision=2
+version=0.31.5
+revision=1
 build_style=cargo
 make_check_args="--bins"
 hostmakedepends="pkg-config"
-makedepends="dbus-devel pulseaudio-devel openssl-devel libsensors-devel"
+makedepends="dbus-devel pulseaudio-devel openssl-devel libsensors-devel pandoc"
```
Comment (mhmdanas) [inline srcpkgs/i3status-rust/template:8]: Why did you add `pandoc` to `makedepends` and not `hostmakedepends`? I'm surprised that this works at all to be honest.

Comment (classabbyamp) [reply srcpkgs/i3status-rust/template:8]: it didn't work. CI "passed" because it ended with pandoc's nocross being triggered and no packages were built

Comment (mhmdanas) [reply srcpkgs/i3status-rust/template:8]: Oh, haha, weird stuff.

@inalone you should put pandoc in hostmakedepends instead of makedepends then.

**Review by Duncaen — CHANGES_REQUESTED** [review]
`pandoc` belongs into `hostmakedepends`.

File: srcpkgs/i3status-rust/template [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
 homepage="https://github.com/greshake/i3status-rust"
 changelog="https://raw.githubusercontent.com/greshake/i3status-rust/master/NEWS.md"
 distfiles="https://github.com/greshake/i3status-rust/archive/refs/tags/v${version}.tar.gz"
-checksum=cd28a90ccb2f9faaaef1e528619f1018981609d77f409abe4877350d810d3324
+checksum=640df4ac85de4efbbc51fd4fa040bfa55270e9c04184f27aca9c427f8d86d3bf
+
+do_build() {
+	cargo auditable build --release --locked --target ${RUST_TARGET}
+	cargo xtask generate-manpage
+}
```
Comment (classabbyamp) [inline srcpkgs/i3status-rust/template:20]:
```suggestion
post_build() {
	cargo xtask generate-manpage
}
```
